<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AliasGame</class>
 <widget class="QWidget" name="AliasGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Игра Псевдоним</string>
  </property>
  <widget class="QLineEdit" name="stonesInput">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Введите количество камней</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Начать игру</string>
   </property>
  </widget>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>360</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Камней на столе: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="takeInput">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Сколько камней взять?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="takeButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Взять камни</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
